analysis/resources/test/valid_dep/Cargo.lock

```diff
@@ -0,0 +1,1512 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Lets remove this file and add a .gitignore so we don't add it back in the future.